docs/migrations.md

```diff
@@ -249,6 +249,36 @@ module.exports = {
     }
 };
 ```
```

Comment:
According to `npm run lint-docs` you must have a blank line before and after any code block.
It is missing one here.